src/style.css

```diff
@@ -1,3 +1,88 @@
+:root {
+  --header-bg-color: #fff;
```

Comment:
Using CSS variables for all the colors, even though it's only required for blue one - nice!